src/TLabel.cpp

```diff
@@ -31,20 +32,18 @@
 
 TLabel::TLabel( QWidget * pW )
 : QLabel( pW )
-, mpHost( 0 )
+, mpHost( Q_NULLPTR )
```

Comment:
I'd rather have this be `mpHost()` instead. Some Qt macros are necessary or useful, this one is neither. :P